PCGen-base/code/src/test/pcgen/base/graph/util/EdgeUtilitiesTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Thomas Parker, 2018.
+ * 
+ * This program is free software; you can redistribute it and/or modify it under the terms
+ * of the GNU Lesser General Public License as published by the Free Software Foundation;
+ * either version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License along with
+ * this library; if not, write to the Free Software Foundation, Inc., 51 Franklin Street,
+ * Fifth Floor, Boston, MA 02110-1301, USA
+ */
+package pcgen.base.graph.util;
+
+import java.util.Arrays;
+import java.util.function.Function;
+
+import junit.framework.TestCase;
+import pcgen.base.graph.base.Edge;
+import pcgen.base.graph.inst.DefaultHyperEdge;
+import pcgen.testsupport.TestSupport;
+
+public class EdgeUtilitiesTest extends TestCase
+{
+
+	public void testConstructor()
+	{
+		TestSupport.invokePrivateConstructor(EdgeUtilities.class);
+	}
+
+	private DefaultHyperEdge<Integer> edge1;
+
+	private Integer node1, node2, node3;
+
+	@Override
+	protected void setUp() throws Exception
+	{
+		node1 = new Integer(3);
+		node2 = new Integer(4);
+		node3 = new Integer(6);
+		edge1 = new DefaultHyperEdge<>(Arrays.asList(new Integer[]{node2, node1, node3}));
+	}
+
+	public void testGetNodeAt()
```

Comment:
Test name could reflect what it does, e.g. testNodesPlacedInGraphAreOnCorrectEdges or something like that.